aten/src/ATen/native/cuda/ScaledGroupMM.cu

```diff
@@ -23,7 +23,6 @@ C10_DIAGNOSTIC_PUSH_AND_IGNORED_IF_DEFINED("-Wunused-but-set-parameter")
 #if defined(BUILD_ROWWISE_FP8_KERNEL)
 
 #include <ATen/native/cuda/GroupMMCommon.cuh>
-#include <ATen/native/cuda/cutlass_utils.cuh>
```

Comment:
Can we kill this yet? Not sure if Meta's internal codebase has upgraded fully yet.